reqhandler/fields.go

```diff
@@ -0,0 +1,33 @@
+package reqhandler
+
+type FieldName string
+
+const (
+	ID       FieldName = "id"
+	QUERY              = "query"
```

Comment:
Actually this is fine. There are cases where iota won't work because of types, but in this case we can use the key as an integer. Thanks!